CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+# Contributing
+
+This is a bespoke fork of the [Pandas Sphinx theme](https://github.com/pandas-dev/pandas-sphinx-theme) use for the various Spyder documentation sites.
+All development ultimately takes place in the ``develop_spyder`` branch.
+While the nominal eventual goal is to merge commonly useful changes downstream and make the rest configurable, the initial choices made after forking the theme will likely make that take an unfeasible amount of effort.
+Nevertheless, any contributions toward that goal are much appreciated.
+
+If you're not comfortable with at least the basics of ``git`` and GitHub, we recommend seeking out a beginner tutorial; if you are already are experienced with it, most of this guide will already be familiar to you.
```

Comment:
Maybe it could be worthy to link here some of the guides that GitHub has:

* Written guides:
    * https://github.com/git-guides/
    * https://guides.github.com/introduction/git-handbook/#basic-git
    * https://guides.github.com/activities/forking/

* Video guides: https://www.youtube.com/githubguides

Reply:
Great suggestions! I added the links inline here, and to the standard contributing guide template that's the baseline for many others in Spyder's and my repos, which will eventually make its way downstream to many other projects when I update them.